File: Challenges/C/ex2.c
#include<stdio.h>
#include<string.h>
int main(int argc,char** argv){
  if(argc!=2){
    printf("Usage: ./prog input\n");
    printf("Hint: ./prog -h\n");
  }else if(strcmp("-h",argv[1])==0){
    printf("A Caesar Cipher is easy to apply to the ascii table.\n");
  }else{
    char psswd[]="BcjuFbtz";
    for(int i=0;i<8;i++){
      if(argv[1][i]+1!=psswd[i]){
        printf("Fail!\n");
        return 0;
      }
    }
    printf("Score!\n");
  }
  return 0;
}

File: Challenges/Go/ex2.go
package main

import "os"
import "fmt"
import "strconv"

func main(){
	if len(os.Args) != 2{
		fmt.Println("Usage: ./prog input")
		fmt.Println("Hint: ./prog -h")
	}else if os.Args[1] == "-h"{
		fmt.Println("This challenge leverages loops to build a comparison string.")
	}else{
    		s := ""
    		for i := 0; i < 10; i++ {
      			s = s + strconv.Itoa(i)
    		}
		if os.Args[1] == s{
			fmt.Println("Score!")
		}else{
			fmt.Println("Fail!")
		}
	}
}
